src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -15,20 +15,13 @@ interface PropsType {
 const MAX_AVATARS_VISIBLE = 2;
 
 export const RoomAttendance: FC<PropsType> = ({ venue, room }) => {
-  const partygoers = usePartygoers();
-  const usersInRoom = useMemo(
-    () =>
-      partygoers?.filter(
-        (partygoer) => partygoer.lastSeenIn?.[`${venue.name}/${room.title}`]
-      ),
-    [partygoers, venue.name, room.title]
-  );
+  const { recentRoomUsers } = useRecentRoomUsers(room.title);
 
-  const numberOfUsersInRoom = usersInRoom?.length;
+  const numberOfUsersInRoom = recentRoomUsers.length;
   if (numberOfUsersInRoom < 1) return <></>;
   return (
     <div className="attendance-avatars">
-      {usersInRoom.map((user, index) => {
+      {recentRoomUsers.map((user, index) => {
         return (
           <div key={`user-avatar-${index}`}>
```

Comment:
Can we use `user.id` here as a unique value? Using `index` in a `key` leads to rendering issues.

```suggestion
      {recentRoomUsers.map((user, index) => {
        return (
          <div key={`user-avatar-${user.id}`}>
```